docs/internal_api.yml

```diff
@@ -348,6 +408,48 @@ definitions:
     example:
       error: "error message"
       request_id: "f7881e82-0492-49fb-b459-795654e7188a"
+  StorageSettings:
+    description: Per tenant storage settings.
+    type: object
+    properties:
+      region:
+        type: string
+        description: AWS region.
+      bucket:
+        type: string
+        description: Bucket name.
+      uri:
+        type: string
+        description: Endpoint URI.
+      key:
+        type: string
+        description: Access key id (for S3 - AWS_ACCESS_KEY_ID).
+      secret:
+        type: string
+        description: Secret access key (for S3 - AWS_SECRET_ACCESS_KEY).
+      token:
+        type: string
+        description: Token.
+      download_expire_seconds:
+        type: integer
+        description: Number of second a presigned download URL is valid.
+      upload_expire_seconds:
+        type: integer
+        description: Number of second a presigned upload URL is valid.
```

Comment:
Let's keep download_expire_seconds and upload_expire_seconds out. I don't think these will need to be customized on a per-tenant level, at least not at this stage.